src/bp/ui-admin/src/Pages/Workspace/Bots/BotItemPipeline.tsx

```diff
@@ -98,6 +102,14 @@ const BotItemPipeline: FC<Props> = ({
             &nbsp;
           </span>
         )}
+        {bot.disabled ? (
+          <span className="bot-name">{bot.name}</span>
+        ) : (
+          <a className="bot-name" href={botStudioLink}>
+            {bot.name}
+          </a>
+        )}
+        {requiresApproval() && <span className="review-needed">Needs your review</span>}
```

Comment:
I know, no design was provided for this but still, we try to use UI components from Blueprint as most as we can